ui/dialogs/trackinfodialog.go

```diff
@@ -58,6 +58,18 @@ func (t *TrackInfoDialog) CreateRenderer() fyne.WidgetRenderer {
 	}
 	c.Add(artists)
 
+	if len(t.track.AlbumArtistNames) > 0 {
+		c.Add(newFormText(lang.L("Album artists"), true))
+		albumArtists := widgets.NewMultiHyperlink()
+		albumArtists.BuildSegments(t.track.AlbumArtistNames, t.track.AlbumArtistIDs)
+		albumArtists.OnTapped = func(id string) {
+			if t.OnNavigateToArtist != nil {
+				t.OnNavigateToArtist(id)
+			}
+		}
+		c.Add(albumArtists)
+	}
```

Comment:
This is just a note for myself - there's a bug below in the composers field - `artists.OnTapped = ...` should be `composers.OnTapped = ...`